2022-08-29 13:54:48,983 INFO werkzeug MainThread :  * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:5000
 * Running on http://10.10.10.235:5000 (Press CTRL+C to quit)
2022-08-29 13:54:51,904 INFO root Thread-1 : Get connection to ('10.100.151.128', 9669)
2022-08-29 13:54:51,921 INFO root Thread-1 : FETCH PROP ON ServiceTemplateDefinition "Jupyter_1" YIELD properties(vertex)
2022-08-29 13:54:51,925 ERROR app Thread-1 : Exception on /cluster_names/Jupyter_1 [GET]
Traceback (most recent call last):
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/flask/app.py", line 2077, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/flask/app.py", line 1525, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/flask/app.py", line 1523, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/flask/app.py", line 1509, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/app/views.py", line 67, in cluster_names
    return render_template("Cluster_name_is_taken.html", cluster_name=varargs[0], result=result)
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/flask/templating.py", line 149, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/jinja2/environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/jinja2/environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/jinja2/environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/jinja2/loaders.py", line 138, in load
    code = environment.compile(source, name, filename)
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/jinja2/environment.py", line 768, in compile
    self.handle_exception(source=source_hint)
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/app/templates/Cluster_name_is_taken.html", line 6, in template
    {% if result}
jinja2.exceptions.TemplateSyntaxError: unexpected '}'
2022-08-29 13:54:51,928 INFO werkzeug Thread-1 : 127.0.0.1 - - [29/Aug/2022 13:54:51] "[35m[1mGET /cluster_names/Jupyter_1 HTTP/1.1[0m" 500 -
2022-08-29 13:55:48,055 INFO root Thread-2 : Get connection to ('10.100.151.128', 9669)
2022-08-29 13:55:48,062 INFO root Thread-2 : FETCH PROP ON ServiceTemplateDefinition "Jupyter_1" YIELD properties(vertex)
2022-08-29 13:55:48,065 INFO werkzeug Thread-2 : 127.0.0.1 - - [29/Aug/2022 13:55:48] "GET /cluster_names/Jupyter_1 HTTP/1.1" 200 -
2022-08-29 13:56:33,616 INFO root Thread-3 : Get connection to ('10.100.151.128', 9669)
2022-08-29 13:56:33,625 INFO root Thread-3 : FETCH PROP ON ServiceTemplateDefinition "Jupyter_1" YIELD properties(vertex)
2022-08-29 13:56:33,626 INFO werkzeug Thread-3 : 127.0.0.1 - - [29/Aug/2022 13:56:33] "GET /cluster_names/Jupyter_1 HTTP/1.1" 200 -
